politeiad/api/v1/v1.go

```diff
@@ -63,24 +64,25 @@ const (
 var (
 	// ErrorStatus converts error status codes to human readable text.
 	ErrorStatus = map[ErrorStatusT]string{
-		ErrorStatusInvalid:                     "invalid status",
-		ErrorStatusInvalidRequestPayload:       "invalid request payload",
-		ErrorStatusInvalidChallenge:            "invalid challenge",
-		ErrorStatusInvalidProposalName:         "invalid proposal name",
-		ErrorStatusInvalidFileDigest:           "invalid file digest",
-		ErrorStatusInvalidBase64:               "corrupt base64 string",
-		ErrorStatusInvalidMIMEType:             "invalid MIME type detected",
-		ErrorStatusUnsupportedMIMEType:         "unsupported MIME type",
-		ErrorStatusInvalidPropStatusTransition: "invalid proposal status transition",
+		ErrorStatusInvalid:                       "invalid status",
+		ErrorStatusInvalidRequestPayload:         "invalid request payload",
+		ErrorStatusInvalidChallenge:              "invalid challenge",
+		ErrorStatusInvalidFilename:               "invalid filename",
+		ErrorStatusInvalidFileDigest:             "invalid file digest",
+		ErrorStatusInvalidBase64:                 "corrupt base64 string",
+		ErrorStatusInvalidMIMEType:               "invalid MIME type detected",
+		ErrorStatusUnsupportedMIMEType:           "unsupported MIME type",
+		ErrorStatusInvalidRecordStatusTransition: "invalid record status transition",
+		ErrorStatusEmpty:                         "no files provided",
```

Comment:
Variable name should be more descriptive, like `ErrorStatusFilesMissing` or something.